pkg/api/azenvtypes.go

```diff
@@ -268,7 +268,7 @@ var (
 		KubernetesSpecConfig: KubernetesSpecConfig{
 			KubernetesImageBase:    "gcr.azk8s.cn/google_containers/",
 			TillerImageBase:        "mcr.microsoft.com/",
-			NVIDIAImageBase:        "dockerhub.azk8s.cn/nvidia/",
+			NVIDIAImageBase:        "mcr.microsoft.com/",
 			AzureCNIImageBase:      "mcr.azk8s.cn/containernetworking/",
```

Comment:
What is mcr.azk8s.cn?  (Just asking, we have not run into a problem with it.

Reply:
I don't know for sure, I would guess it is a (no longer maintained?) mirror of certain mcr.microsoft.com k8s images?